core/Rename.hpp

```diff
@@ -109,4 +137,40 @@ namespace olympia
         ScoreboardFactories sb_facts_;
         ScoreboardTreeNodes sb_tns_;
     };
+
+    // Rename Tester Class
+    class RenameTester
```

Comment:
Consider moving this class to the Rename test.

Reply:
Moved to Rename_test.cpp in [128fab2](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/128fab23c955b2226bb1f60a28a32a5d96241817)